common/src/main/java/com/wynntils/features/utilities/TranslationFeature.java

```diff
@@ -71,7 +71,10 @@ public void onChat(ChatMessageEvent.Match e) {
                         // We failed to get a translation; send the original message so it's not lost
                         messageToSend = originalMessage;
                     }
-                    McUtils.mc().doRunTask(() -> McUtils.sendMessageToClient(messageToSend.getComponent()));
+                    McUtils.mc().doRunTask(() -> {
+                        // We need this to not trigger any events so don't use McUtils.sendMessageToClient
```

Comment:
This will probably not work with chat tabs. It's also unclear what the expected behavior should be with chat tabs..?